ctapipe/instrument/camera/geometry.py

```diff
@@ -623,12 +636,15 @@ def rotate(self, angle):
             rotation angle with unit (e.g. 12 * u.deg), or "12d"
 
         """
+        angle = Angle(angle)
```

Comment:
Maybe also allow an `Angle` here and avoid the conversion in that case?

Reply:
Rotate is not called often (not e.g. in every event)